### PR TITLE
added zip to workspace container (so there is no composer warning)

### PR DESCRIPTION
**Pull request by vladyslavstartsev**
Closing this one https://github.com/laradock/laradock/issues/1870
##### I completed the 3 steps below:

- [x] I've read the [Contribution Guide](http://laradock.io/contributing).
- [x] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [x] I enjoyed my time contributing and making developer's life easier :)

### CODE REVIEW

File: workspace/Dockerfile [hunk 1]
```diff
@@ -174,6 +174,19 @@ RUN if [ ${INSTALL_DRUSH} = true ]; then \
...
+###########################################################################
+# Zip:
+###########################################################################
+
+USER root
+
+ARG INSTALL_ZIP=false
+
+RUN if [ ${INSTALL_ZIP} = true ]; then \
+  # Install the PHP SSH2 extension
```
Comment (lanphan) [inline workspace/Dockerfile]: would you please correct this comment?

Comment (vladyslavstartsev) [reply workspace/Dockerfile]: Yes! Thanks a lot

**Comment by lanphan** [comment]
As this is required by composer command, should we move installing zip + unzip to https://github.com/laradock/workspace ?

In addition, would you please remove all existence of installing unzip + zip in this Dockerfile, for ex, if INSTALL_TERRAFORM = true, it'll install unzip?

**Comment by vladyslavstartsev** [comment]
@bestlong what do you think? leave it here or move to https://github.com/laradock/workspace ?

**Comment by bestlong** [comment]
https://github.com/laradock/php-fpm/issues/3

i think move to workspace and install by default is grate.

**Comment by bestlong** [comment]
but there is a big problem. docker hub not auto build new image both workspace and php-fpm projects.

**Comment by vladyslavstartsev** [comment]
> but there is a big problem. docker hub not auto build new image both workspace and php-fpm projects.

so... what should I do in my turn?

**Comment by bestlong** [comment]
Direct installation in laradock/laradock, no need WORKSPACE_INSTALL_ZIP.

File: php-fpm/Dockerfile [hunk 2]
```diff
@@ -47,7 +55,7 @@ RUN apt-get update -yqq && \
 # Optional Software's will only be installed if you set them to `true`
 # in the `docker-compose.yml` before the build.
 # Example:
-#   - INSTALL_ZIP_ARCHIVE=true
```
Comment (vladyslavstartsev) [inline php-fpm/Dockerfile]: changed the comment to some other env var because if we will leave it like this it will mislead people

**Pull request merged**